### PR TITLE
[CAL-817] Filter Projects.list on participant_id and status

### PR DESCRIPTION
**Pull request by VerstraeteBert**
Allowing users to filter Projects.list output on participant_id and project status

### CODE REVIEW

File: src/07-projects/projects.apib [hunk 1]
```diff
@@ -11,6 +11,14 @@ Get a list of projects.
 + Request (application/json;charset=utf-8)
 
     + Attributes (object)
+        + filter (object, optional)
+            + status (enum[string], optional)
+                + Members
+                    + active
```
Comment (WouterSioen) [inline src/07-projects/projects.apib:17]: Isn't active a bit of a weird term? Sounds to me like this is the status during the meeting and that it means the meeting is happening at the moment.

Comment (VerstraeteBert) [reply src/07-projects/projects.apib]: You're right, a name like `planning_status` would make more sense here, it has already implemented with `status` for a while but I can easily change it. Any other suggestions?

Comment (adognameddog) [reply src/07-projects/projects.apib:17]: @WouterSioen this is the status of a project, not of a meeting. There are alternatives for `active`, like `busy`, `ongoing`, `current`, but I think `active` is the best one.

Comment (WouterSioen) [reply src/07-projects/projects.apib:17]: Then it makes sense, not sure where I got the idea that this was for meetings.

**Review by WouterSioen — APPROVED** [review]

**Review by psrpinto — APPROVED** [review]

**Review by adognameddog — APPROVED** [review]

**Pull request merged**